src/elements/content-sidebar/ActivitySidebar.js

```diff
@@ -557,7 +559,15 @@ class ActivitySidebar extends React.PureComponent<Props, State> {
     };
 
     render() {
-        const { elementId, file, isDisabled = false, onVersionHistoryClick, getUserProfileUrl } = this.props;
+        const {
+            elementId,
+            file,
+            isDisabled = false,
```

Comment:
I think this default is handled by `defaultProps` above, right?

Reply:
It is defined by defaultProps, but when removed from render, ActivitySidebar fails to load. Not why this happens